CMakeLists.txt

```diff
@@ -575,158 +664,50 @@ if(GRIDDYN_GENERATE_DOXYGEN_DOC)
             VERBATIM
         )
     endif(DOXYGEN_FOUND)
-endif(GRIDDYN_GENERATE_DOXYGEN_DOC)
-
-# ------------------------------------------------------------
-# Enable HELICS executable
-# ------------------------------------------------------------
-
-option(HELICS_EXECUTABLE "Enable the HELICS executable to be built" OFF)
-# If HELICS_EXECUTABLE is load the helics directory
-# link against them.  the variable is here since we would enable the zmq libraries with it on and that should be done
-# before actually including helics
-
-if(HELICS_EXECUTABLE)
-    show_variable(
-        HELICS_INSTALL_PATH
-        PATH
-        "path to the helics installation"
-        "${PROJECT_BINARY_DIR}/libs"
-    )
-    set(ZMQ_NEEDED ON CACHE BOOL "ZMQ is needed" FORCE)
-else(HELICS_EXECUTABLE)
-    hide_variable(HELICS_INSTALL_PATH)
-endif(HELICS_EXECUTABLE)
-
-# ------------------------------------------------------------
-# Enable ZMQ interfacing
-# ------------------------------------------------------------
-
-option(
-    ENABLE_NETWORKING_LIBRARY "Enable network based communiation components" OFF
-)
-
-cmake_dependent_option(
-    DIME_ENABLE
-    "Enable connection with Dime"
-    OFF
-    "ENABLE_NETWORKING_LIBRARY"
-    OFF
-)
-cmake_dependent_option(
-    TCP_ENABLE
-    "Enable TCP connection library"
-    ON
-    "ENABLE_NETWORKING_LIBRARY"
-    OFF
-)
-if(DIME_ENABLE)
-    set(ZMQ_NEEDED ON CACHE BOOL "ZMQ is needed" FORCE)
-endif(DIME_ENABLE)
-
-hide_variable(ZMQ_NEEDED)
-
-cmake_dependent_option(
-    ZMQ_ENABLE
-    "Enable ZMQ networking library"
-    OFF
-    "NOT ZMQ_NEEDED"
-    OFF
-)
-
-# If ZMQ library is enabled try to locate it and link against it
-
-if((ZMQ_ENABLE) OR (ZMQ_NEEDED))
-    include(addZeroMQ)
-    if(NOT ZeroMQ_FOUND)
-        message(SEND_ERROR "unable to locate zmq library")
-    endif()
-endif()
+endif(ENABLE_GRIDDYN_DOXYGEN)
 
-if(ENABLE_NETWORKING_LIBRARY)
-    add_subdirectory(src/networking)
-    target_link_libraries(griddyn_optional INTERFACE networking)
-endif()
-
-if(HELICS_EXECUTABLE)
-    include(addHELICS)
-    add_subdirectory(src/helics)
-    target_link_libraries(griddyn_optional INTERFACE helicsLib)
-endif(HELICS_EXECUTABLE)
 
+# TODO ???
```

Comment:
this is mainly for windows to enable running the debugger since it may have to copy some shared libraries to various folders to make things run. These files are in the bin directory.